semver/__init__.py

```diff
@@ -59,7 +61,7 @@ def get_branches(self):
         message = str(p.stdout.read())
         branch = b.stdout.read().decode('utf-8').rstrip()
         logger.info('Main branch is ' + branch)
-        matches = GET_COMMIT_MESSAGE.search(message)
+        matches = GET_COMMIT_MESSAGE.search(message.replace('\\n','\n').replace('\\',''))
```

Comment:
Both types of quotes cause problems

Reply:
I was able to test this locally with both double and single quotes.

Reply:
The reason I replace the `\\` is because in the string all quotes are `\'` or `\"`. I'm thinking maybe an update with a dependency?

Reply:
ok i think i get it